common/common.go

```diff
@@ -87,3 +93,32 @@ func SwapNibbles(k []byte) []byte {
 	}
 	return result
 }
+
+// ToTOML encodes a state type into a TOML file.
+// It uses the BurntSuchi TOML parser.
```

Comment:
BurntSushi ???

Reply:
Yes lol its a really good TOML parser

Reply:
nice ahah it's mispelled in this comment tho